lib/profileAgents.js

```diff
@@ -62,10 +63,15 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
  * @param {string} [profileId] - The ID of a profile.
  * @param {string} [accountId] - The ID of an account.
  * @param {string} [token] - An application token.
+ * @param {string} [didMethod='key'] - Profile ID DID method (key or v1, etc).
+ * @param {string} [didMethodMode='test'] - Optional DID Method ledger mode
+ *   (relevant only for Veres One for the moment, to select testnet or main).
  *
  * @return {Promise<Object>} resolves to a ProfileAgent.
  */
-exports.create = async ({accountId, profileId, token} = {}) => {
+exports.create = async ({
+  accountId, profileId, token, didMethod = 'key',
+  didMethodMode = 'test'} = {}) => {
```

Comment:
Can we make this `didMethodOptions` instead with a `mode` in it? We don't know what other options a particular did method might need and this will give us more flexibility for consolidating them.

Reply:
Please apply same in other places where relevant.

Reply:
Changed, thanks.